node/node.go

```diff
@@ -452,42 +410,30 @@ func (node *Node) Start() error {
 							}
 							errChan <- ctx.Err()
 						default:
+							payload := msg.GetData()
 							node.HandleMessage(
 								payload[p2pMsgPrefixSize:], msg.GetFrom(),
 							)
 						}
 					}
 				}()
-
 			}
 		}()
 
 		go func() {
-			slowDown, coolDown := false, throttle
 
 			for {
-
-				select {
-				case s := <-needThrottle:
-					slowDown = true
-					coolDown = s
-					utils.Logger().Info().
-						Dur("throttle-delay-miliseconds", s.Round(time.Millisecond)).
-						Msg("throttle needed on acceptance of messages")
-				case <-releaseThrottle:
-					utils.Logger().Info().Msg("p2p throttle released")
-					slowDown = false
-				default:
-					if slowDown {
-						<-time.After(coolDown)
-					}
-				}
-
 				nextMsg, err := sub.Next(context.Background())
 				if err != nil {
 					errChan <- err
 					continue
 				}
+
+				payload := nextMsg.GetData()
+				if len(payload) < p2pMsgPrefixSize {
```

Comment:
print a warning message here?

Reply:
It’s would a very noisy log I think, and this part is existing logic from many months ago too.